services/endpoint/endpoint.tester.js

```diff
@@ -146,7 +146,10 @@ t.create('Invalid schema (mocked)')
         schemaVersion: -1,
       })
   )
-  .expectJSON({ name: 'custom badge', value: 'invalid response data' })
+  .expectJSON({
+    name: 'custom badge',
+    value: 'invalid properties: schemaVersion',
```

Comment:
I'm a little confused in terminology differences (`properties` vs `parameters` vs `keys`)? Are they all synonymous and if not which is which in this context?

Reply:
"Keys" is the name that's internal to the code. **properties** seems like the friendliest way to explain to the endpoint provider that the object they're returning doesn't have the right stuff in it. I think it's a little clearer than keys.

I think we use the word "parameters" for query parameters, which makes sense.

Reply:
Ahh okay, i forgot this was the reverse end. it would be invalid/unexpected properties in the response object the endoint was returning that would trigger that right?

Reply:
Yea, correct. In this case `schemaVersion` is set to an invalid value.

Reply:
Thanks, I'm up to speed now 👍